components/locale_core/src/langid.rs

```diff
@@ -76,6 +73,38 @@ use alloc::borrow::Cow;
 /// assert_eq!(li.variants.get(0), Some(&variant!("valencia")));
 /// ```
 ///
+/// # Ordering
+///
```

Comment:
Suggestion: move this example to the `total_cmp` function and we can make that the landing page for a more detailed review of how ordering works.